setup.cfg

```diff
@@ -11,4 +11,4 @@ install_requires =
     nose >= 1.3.7
     lxml >= 4.4.1
     matplotlib > 3
-    assimulo >= 3.2
\ No newline at end of file
+    assimulo >= 3.3
```

Comment:
Good! Perhaps we should even update README such that the dependency on Assimulo is directly visible on the front page? Or we can create a separate issue for that.